N/NOMAD/build_tarballs.jl

```diff
@@ -36,8 +37,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", 
+    uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
 ]
-
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"8.1.0")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"12", julia_compat="v1.8")
```

Comment:
Why using GCC 12 and Julia v1.8?  There is no way this is going to work with GCC 12

Reply:
why not?

Reply:
i've built this with gcc 12/11 a bunch of times

Reply:
By "work" I mean that it won't work at runtime,  because we need to use libstdc++ from Julia, which at the moment has GCC up to version 11 (and older versions of Julia will have older versions of libstdc++).  For maximum compatibility you _**must**_ use the oldest version of GCC possible, not the latest one.

Reply:
the only reason it's gcc 8.1.0 is that the code errors that it's not tested on gcc 8.1.0

Reply:
i see